cypress/page-objects/loginAngular.js

```diff
@@ -0,0 +1,12 @@
+
+import { VAR } from '../const/constants';
+
+export const IDs = {
```

Comment:
Not all here are IDs, but no use another name like `selectors`?

Reply:
Change in the branch "NewArticule"